ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Welcome to the issues section if it's your first time!
```

Comment:
You mean, comment so that one creating an issue do not post this text accidentally?